pages_bots/mdwiki_page.py

```
@@ -0,0 +1,94 @@
+purge       = page.purge()
+'''
+"""
+# ---
+import os
+import sys
+# ---
+if "mwclient" not in sys.argv:
+    sys.argv.append("nomwclient")
+    print("sys.argv.append('nomwclient')")
+
+from ..super.S_API import bot_api
```

Pylint (3.3.7)

[convention] Import "from ..super.S_API import bot_api" should be placed at the top of the module

(C0413)